File: codeabb/C/p9_triangles.c
// codeabbey, problem 9
// solution by whoisrgj

#include <stdio.h>

int main(int argc, char *argv[]) {
    int N, i, a, b, c, is_tri;
    
    fscanf(stdin, "%d", &N);
    for(i = 0; i < N; i++) {
        is_tri = 0;
        fscanf(stdin, "%d %d %d", &a, &b, &c);
        if(((a+b) >= c) && ((a+c) >= b) && ((b+c) >= a))
            is_tri = 1;
        fprintf(stdout, "%d ", is_tri);
    }
    return 0;
}

File: codeabb/Python/p9_triangles.py
"""
CodeAbbey, Problem 9
Coded by whoisrgj
"""


def is_triangle(a, b, c):
    return a+b >= c and a+c >= b and b+c >= a


def main():
    n = int(input())
    results = []
    for _ in range(n):
        a, b, c = map(int, input().split())
        results.append(int(is_triangle(a, b, c)))
    print(*results)

if __name__ == '__main__':
    main()
